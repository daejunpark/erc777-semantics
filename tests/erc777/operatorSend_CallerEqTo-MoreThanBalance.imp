test.preamble();

// Testing operatorSend(From,To,Value,"","") when From = caller
// Throw because Value > BalanceFrom.

test.makeCaller(9);           // switch to 9
test.operatorSend(7, 9, 71, "", "");  // send 71 from 7 to 9; balance of 7 is 70
