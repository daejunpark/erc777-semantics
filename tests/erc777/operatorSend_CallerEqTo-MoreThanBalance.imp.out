Changing the caller to 9
9 is sending 71 from 7 to 9 (data:""),  (operatorData:"")
REVERT
Balance of 7 is 70
Balance of 9 is 90
